### PR TITLE
Spine service mock PR

### PR DESCRIPTION
**Pull request by awilkins**

### CODE REVIEW

File: acceptance/User.Specs/Hooks/Screenshot.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+using TechTalk.SpecFlow;
+
+using OpenQA.Selenium;
+
+using GpConnect.DataSharing.User.Specs.Drivers;
+
+namespace GpConnect.DataSharing.User.Specs.Hooks
+{
+
```
Comment (leew) [inline acceptance/User.Specs/Hooks/Screenshot.cs:9]: Extra white space

File: acceptance/User.Specs/Hooks/Screenshot.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    string Runname = title + DateTime.Now.ToString("yyyy-MM-dd-HH_mm_ss.jpg");
+
+                    string reportFolder = "./reports";
+                    var d = Directory.CreateDirectory(reportFolder);
+                    ss.SaveAsFile("./reports/" + Runname, ScreenshotImageFormat.Jpeg);
+                }
+            }
+            catch
+            {
+                //Console.WriteLine("catch");
```
Comment (leew) [inline acceptance/User.Specs/Hooks/Screenshot.cs]: Commented Code

File: acceptance/User.Specs/Hooks/Screenshot.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            }
+            finally
+            {
+                if (_browser != null)
+                {
+                    _browser.Quit();
+                    _browser = null;
+
```
Comment (leew) [inline acceptance/User.Specs/Hooks/Screenshot.cs]: w/s

File: acceptance/User.Specs/PageObjects/ApplicationLandingPageObject.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            _webDriver.Url = URL(PATH);
+        }
+
+        public bool IsStartButtonVisible()
+        {
+            var wait = new WebDriverWait(_webDriver, DefaultWait);
+            return wait.Until(driver => IsButtonVisible("Apply Now"));
+        }
+
+        // public void ClickStart(){
```
Comment (leew) [inline acceptance/User.Specs/PageObjects/ApplicationLandingPageObject.cs]: commented code

File: acceptance/User.Specs/Steps/ApplicationLandingStepDefinitions.cs [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            _applicationLandingPage.Open();
+        }
+
+        [Then(@"A button to start the application is present")]
+        public void ThenAButtonToStartTheApplicationSearchIsPresent()
+        {
+            Assert.True(_applicationLandingPage.IsStartButtonVisible());
+        }
+
+        // [Then(@"the support email address is present")]
```
Comment (leew) [inline acceptance/User.Specs/Steps/ApplicationLandingStepDefinitions.cs]: commented Code

File: acceptance/User.Specs/Steps/TransparencyLandingStepDefinitions.cs [hunk 6]
```diff
@@ -34,23 +34,11 @@ public void ThenAButtonToStartTheTransparencySearchIsPresent()
...
-        {
-            Assert.True(_transparencyLandingPage.SupportPhone.Displayed);
-        }
-
-        [When(@"I click the start now button")]
-        public void WhenIClickTheStartNowButton()
-        {
-            _transparencyLandingPage.ClickStart();
-        }
+        // [When(@"I click the start now button")]
```
Comment (leew) [inline acceptance/User.Specs/Steps/TransparencyLandingStepDefinitions.cs]: Commented Code

**Review by leew — COMMENTED** [review]
Small whitespace and commented code changes

**Comment by awilkins** [comment]
Addressed issues from review.

**Pull request merged**